crates/cli/src/errors.rs

```diff
@@ -26,7 +26,7 @@ pub enum CliError {
     ConfigType {
         key: String,
         kind: &'static str,
-        found: toml_edit::Item,
+        found: Box<toml_edit::Item>,
```

Comment:
Is the boxing necessary now because of the version upgrade?

Reply:
Yeah, it was giving a lint about a too-big error variant (261 bytes :grimacing:). The easiest way was to just box it, and now it's 40 bytes.

Reply:
Cool thank you :+1: